### DISCUSSION TITLE
My selected school has incorrect or incomplete information e.g. domains or campus location

### DISCUSSION THREAD

**Discussion by yannisctr** [community/community#140279] [GitHub Education] [answered]
### Hi everyone,

Hi, today I wanted to redeem the GitHub student pack. all my info was good except for the campus location. I explain. The main campus is based at Cergy, Near Paris, but they have another campus at Pau and I study at the Pau campus. But my mail address is linked to the main campus, which is normal, but i'm located to Pau so GitHub think that I am not on the campus. https://plan.cyu.fr/cartographie map from the official website showing at bottom left corner of France the Pau campus. How can i solve this issue ? sorry for my English :).

**Comment by davevad93** [comment, marked as answer]
Hi @yannisctr ,

I suggest you to contact support by opening a ticket [here](https://support.github.com/contact/education), select "`My selected school has incorrect or incomplete information e.g. domains or campus location`" and explain your issue.

**Reply by yannisctr** [reply]
thanks !! :)

**Reply by davevad93** [reply]
You're welcome @yannisctr 👍
